changelog/chkprintf.md

```diff
@@ -0,0 +1,51 @@
+Check that arguments to a printf format string are compatible
+with that string. Issue errors for incompatibilities.
```

Comment:
This is a title and so two sentences are far too long.

Also it doesn't show up in [preview](http://dtest.dlang.io/artifact/website-fd8ec731d07accf437306462d3566c360e8bfabe-fd810bf46817b5113d3f14e510dc6142/web/changelog/pending.html). @wilzbach do we support `md` file ? [Our documentation](https://github.com/dlang/dmd/blob/master/changelog/README.md) only says `.dd`.

Reply:
urgh, it'll take a couple hours to go all green again if I modify the changelog :-(

Reply:
Yeah while DMD now supports markdown, the changelog script hasn't been updated yet.